test/system/500-networking.bats

```diff
@@ -88,6 +88,7 @@ load helpers
 
 # Issue #5466 - port-forwarding doesn't work with this option and -d
 @test "podman networking: port with --userns=keep-id" {
+    skip_if_not_rootless "--userns=keep-id only works in rootless mode"
```

Comment:
Can you change the keep-id for some other namespace that works both with root and rootless? I guess a custom --uidmap should work? I think it is important to run this test as root as well.

Reply:
Let's open a PR to add a test for --userns=auto.